src/app/shared/certificate-verification-popover/certificate-verification-popover.component.html

```diff
@@ -10,18 +10,18 @@
     <img src="../../../assets/icons/badge.svg" alt="Certificate Icon">
   </div>
   <div class="certificate-details">
-    <div class="certificate-issued-to">Certificate issued to</div>
+    <div class="certificate-issued-to">{{"CERTIFICATE_ISSUED"|translate}}</div>
     <div class="certificate-recipient">{{ data.userName }}</div>
-    <div class="certificate-message">For successfully completing the project</div>
+    <div class="certificate-message">{{"CERTIFICATE_SUCCESS"|translate}}</div>
     <div class="certificate-project">{{ data.projectName }}</div>
     <div class="certificate-date">
-      On Elevate on {{ data.completedDate | date: 'dd/MM/yyyy' }}
+      {{"CERTIFICATE_DATE_ON" | translate}} {{ data.completedDate | date: 'dd/MM/yyyy' }}
```

Comment:
Try passing the date also to the en.json file dynamically along with the string
Change the key to "CERTIFICATE_COMPLETION_DATE"